pkg/trace/README.md

```diff
@@ -1,136 +1,86 @@
-# trace: push arbitrary trace level data to an influxdb instance
+# trace package
 
-This package has code to create a client that can be used to push events to an
-influxdb instance. It is used to collect trace data from many different nodes in
-a network. If there is no URL in the config.toml, then the underlying client is
-nil and no points will be written. The provided chainID and nodeID are used to
-tag all points. The underlying client is exposed to allow for custom writes, but
-the WritePoint method should be used for most cases, as it enforces the schema.
+The `trace` package provides a decently fast way to store traces locally.
 
-## Usage and Schema
+## Usage
 
-To use this package, first create a new client using the `NewClient` function,
-then pass that client to the relevant components that need to push events. After
-that, you can use the `WritePoint` method to push events to influxdb. In the below
-example, we're pushing a point in the consensus reactor to measure exactly when
-each step of consensus is reached for each node.
+To enable the local tracer, add the following to the config.toml file:
 
-```go
-client.WritePoint(RoundStateTable, map[string]interface{}{
-		HeightFieldKey: height,
-		RoundFieldKey:  round,
-		StepFieldKey:   step.String(),
-})
-```
-
-Using this method enforces the typical schema, where we are tagging (aka
-indexing) each point by the chain-id and the node-id, then adding the local time
-of the creation of the event. If you need to push a custom point, you can use
-the underlying client directly. See `influxdb2.WriteAPI` for more details.
-
-### Schema
-
-All points in influxdb are divided into a key value pair per field. These kvs
-are indexed first by a "measurement", which is used as a "table" in other dbs.
-Additional indexes can also be added, we're using the chain-id and node-id here.
-This allows for us to quickly query for trace data for a specific chain and/or
-node.
-
-```flux
-from(bucket: "e2e")
-  |> range(start: -1h)
-  |> filter(
-    fn: (r) => r["_measurement"] == "consensus_round_state"
-      and r.chain_id == "ci-YREG8X"
-      and r.node_id == "0b529c309608172a29c49979394734260b42acfb"
-    )
-```
+```toml
+# The tracer to use for collecting trace data.
+trace_type = "local"
 
-We can easily retrieve all fields in a relatively standard table format by using
-the pivot `fluxQL` command.
+# The size of the batches that are sent to the database.
+trace_push_batch_size = 1000
 
-```flux
-from(bucket: "mocha")
-  |> range(start: -1h)
-  |> filter(fn: (r) => r._measurement == "consensus_round_state")
-  |> pivot(rowKey:["_time"], columnKey: ["_field"], valueColumn: "_value")
+# The list of tables that are updated when tracing. All available tables and
+# their schema can be found in the pkg/trace/schema package. It is represented as a
+# comma separate string. For example: "consensus_round_state,mempool_tx".
+tracing_tables = "consensus_round_state,mempool_tx"
```

Comment:
[nit] Based on my testing, this field automatically gets populated with the names of all available tables hence, I suppose there is no need for modifications unless there's a specific desire to exclude certain tables from the tracing process. If that's the case, it would indeed be helpful to clarify that it is possible to selectively skip tracing specific tables.

Reply:
thanks for not blocking on this

I think I'm having some author blindness here :sweat_smile:, so I'll defer to you on this. are you refering to the readme, or do you think the first sentence here doesn't describe that? if its the first sentence, do you think 

> The list of tables that are updated when tracing. Tables not included below will not be traced.

works better?